### DISCUSSION TITLE
Another Face Correction Post

### DISCUSSION THREAD

**Discussion by Kaztigo** [lshqqytiger/stable-diffusion-webui-directml#195] [Q&A]
I was trying to make faces look nice, but the inpaint_only+lama (wich I was using before with A!detailer) crashes the image generation. What other options or suggestion can you give me if I want to correct faces. I cant really use upscale a lot because I run out of  vram.
Currently I have Ryzen 7 5800x, with a Radeon 6600 xt

**Comment by songib** [comment]
I just use Adetailer without any ctrlnet model. But when I use it, it causes a memory leak on my 8GB card. before recent update was working fine, idk.

**Reply by Kaztigo** [reply]
what wa your work flow with Adetailer? what options did you select for the face correction?

**Reply by songib** [reply]
For the face, I just used 0.25 or 0.35, and for the other part, I used 0.4 Denoising strength, which you can adjust to your liking.
Model 1st is person, 2nd is hand, and 3rd is face, so it will always be in that step. and for hand, I put "(Beautiful Perfect Hand, Detailed:1.3)" in the positive prompt, so it's not making face or feet. for other I just leave it empty, so it will use the main prompt.
